src/app/account/repo/checks/[id]/form.js

```diff
@@ -19,31 +22,41 @@ export default function Form({ id }) {
 }
 
 export function FormBadge({ src }) {
+  const [copy, setCopy] = useState(false);
+  const copyHandle = async () => {
+    const url = `![HealthCheck](${src})`;
+    await navigator.clipboard.writeText(url);
+    setCoppy(true);
```

Comment:
This is spelt incorrectly and not matching the set function `setCopy`

```suggestion
    setCopy(true);
```